llvm/lib/Target/AMDGPU/GCNSubtarget.h

```diff
@@ -1676,11 +1676,13 @@ class GCNUserSGPRUsageInfo {
 
   bool hasPrivateSegmentSize() const { return PrivateSegmentSize; }
 
+  bool hasLDSKernelId() const { return LDSKernelId; }
```

Comment:
I don't know why this is defined in GCNSubtarget.h, it's not related to the subtarget